demos/common/python/models/detection_model.py

```diff
@@ -0,0 +1,76 @@
+from .image_model import ImageModel
+from .utils import Detection, load_labels, clip_detections
+
+
+class DetectionModel(ImageModel):
+    def __init__(self, ie, model_path, input_transform=None, resize_type='default', 
```

Comment:
If we'll implement my suggestion anbout postprocess, resize_type might be moved to preprocess() function parameters and self.resize_type might be deleted. Thischange is still discussible.

Reply:
I don't get it. How? The `resize_type` is an `ImageModel` parameter.

Reply:
No, thats bad idea. We need to know the preprocessor resizer for postprocessor. As possible solution - pass it in meta data